src/lib/y2storage/md_container.rb

```diff
@@ -64,8 +64,17 @@ def software_defined?
 
   protected
 
+    # Objects of this class are not considered RAIDs at all
     def types_for_is
-      super << :md_container
+      types = super
+
+      types.delete(:disk_device)
+      types.delete(:raid)
+      types.delete(:bios_raid)
+      types.delete(:software_raid)
```

Comment:
are you sure about this change? It looks a bit agresive change here.

Reply:
Yes, it ensures the same behavior than before.